package.json

```diff
@@ -30,19 +30,19 @@
     "d3-shape": "^1.0.0",
     "d3-transition": "^1.0.0",
     "dateformat": "^2.0.0",
-    "dompurify": "^1.0.0",
+    "dompurify": "^1.0.2",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.1",
     "gulp": "^3.9.1",
     "hammerjs": "^2.0.6",
-    "html-to-react": "^1.2.2",
+    "html-to-react": "1.2.12",
```

Comment:
Just curious: why do we need 1.2.12 exactly?

Reply:
To avoid the React 16 peer dependency issue:
![image](https://user-images.githubusercontent.com/13863530/31978594-d768814a-b98d-11e7-8d57-b847be9ba894.png)

Reply:
Another option would be to use React 16. It would probably only take ~1-2 days to upgrade though, I think most community packages should support it by now. And they didn't introduce async rendering this version, which I'm pretty sure will break our observable mixin if nothing else.

React 16 error boundaries make it easier to improve developer experience for crafting feature info templates as well.

Reply:
We should definitely do that at some point, but no major hurry.  In any case, that sounds like a separate PR.